lib/trilogy_adapter/lost_connection_exception_translator.rb

```diff
@@ -45,7 +45,7 @@ def translate_ruby_exception
       end
 
       def translate_trilogy_exception
-        return unless exception.is_a?(Trilogy::Error)
+        return unless exception.is_a?(Trilogy::BaseError)
```

Comment:
`Trilogy::BaseError` includes `Trilogy::Error`, so this one was OK as is. I think that, given the specific error messages we handle, we could narrow this to `exception.is_a?(Trilogy::QueryError)`, but I'm not 100% sure so it might be easier to leave this one alone.

Reply:
Sounds well.

Thanks!